### PR TITLE
doc: mailmap for v12.0.2 

### PR DESCRIPTION
**Pull request by theanalyst**
new affiliations and a few corrections

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -206,6 +212,7 @@ Jonas Keidel <jonas@jonas-keidel.de>
 Jordan Dorne <jordan.dorne@gmail.com>
 Josh Durgin <jdurgin@redhat.com>
 Josh Durgin <jdurgin@redhat.com> <joshd@redhat.com>
+Josh Durgin <jdurgin@redhat.com> <jduring@redhat.com>
```
Comment (dachary) [inline .mailmap]: :-)

File: .organizationmap [hunk 2]
```diff
@@ -253,6 +257,7 @@ Intel <contact@intel.com> Wang, Yaguang <yaguang.wang@intel.com>
 Intel <contact@intel.com> Xiaoxi Chen <xiaoxi.chen@intel.com>
 Intel <contact@intel.com> Yan, Zheng <zheng.z.yan@intel.com>
 Intel <contact@intel.com> Yuan Zhou <yuan.zhou@intel.com>
+Intel <contact@intel.com> Ziye Yang <ziye.yang@intel.com>
```
Comment (dachary) [inline .organizationmap]: duplicate

File: .organizationmap [hunk 3]
```diff
@@ -682,6 +694,7 @@ Unaffiliated <no@organization.net> Xingyi Wu <wuxingyi2015@outlook.com>
 Unaffiliated <no@organization.net> Xinze Chi <xmdxcxz@gmail.com>
 Unaffiliated <no@organization.net> Xiong Yiliang <xiongyiliang@xunlei.com>
 Unaffiliated <no@organization.net> Xu Biao <xubiao.codeyz@gmail.com>
+Unaffiliated <no@organization.net> Yao Zongyou <yaozongyou@vip.qq.com>
```
Comment (dachary) [inline .organizationmap]: after Yannick

File: .organizationmap (hunk 2 above)
Comment (theanalyst) [inline .organizationmap]: ah

File: .organizationmap (hunk 3 above)
Comment (theanalyst) [inline .organizationmap]: ack

File: .organizationmap [hunk 4]
```diff
@@ -333,6 +333,7 @@ Pacific Northwest National Laboratory <contact@pnl.gov> Evan Felix <evan.felix@p
 Pacific Northwest National Laboratory <contact@pnl.gov> Scott Devoid <devoid@anl.gov>
 Piston Cloud Computing <info@pistoncloud.com> Mike Lundy <mike@fluffypenguin.org>
 Pogoapp <contact@pogoapp.com> Paul Meserve <paul@pogodan.com>
+Proxmox Server Solutions GmbH <contcat@proxmox.com> Fabian Grünbichler <f.gruenbichler@proxmox.com>
```
Comment (Fabian-Gruenbichler) [inline .organizationmap]: There is a typo here ("contcat" vs "contact"), but if possible, "office@proxmox.com" would be the more appropriate organizational address.

Thanks!

Comment (theanalyst) [inline .organizationmap]: Thanks, will update

**Comment by theanalyst** [comment]
jenkins test this please

**Comment by liewegas** [comment]
needs rebase

**Comment by theanalyst** [comment]
@dachary rebased, is this ok to merge

**Pull request merged**